docs/contributing/upgrading-dependencies.md

```diff
@@ -3,6 +3,22 @@
 This document outlines how we manage the dependencies that GitHub Desktop needs,
 as some need to be handled differently to others
 
+## General Guidelines
+
+In the interest of stability and caution we tend to stay a version (or more) behind on our most critical dependencies. We vary whether this is a major, minor, or patch version depending on the development practices of the dependency's project.
+
+| Dependency  | Versions Behind Latest |
+| --- | --- |
+| electron | >= 1 major |
+| electron-builder | >= 1 major |
```

Comment:
This project hasn't seen a major update in a while

- we're on `20.28.4` which was published `Sep 5 2018`
- latest is `20.39.0` which was published `Feb 26 2019`

```suggestion
| electron-builder | >= 1 minor |
```